cf-conventions.adoc

```diff
@@ -1,13 +1,36 @@
 include::version.adoc[]
 = NetCDF Climate and Forecast (CF) Metadata Conventions
-Brian{nbsp}Eaton; Jonathan{nbsp}Gregory; Bob{nbsp}Drach; Karl{nbsp}Taylor; Steve{nbsp}Hankin; Jon{nbsp}Blower; John{nbsp}Caron; Rich{nbsp}Signell; Phil{nbsp}Bentley; Greg{nbsp}Rappa; Heinke{nbsp}Höck; Alison{nbsp}Pamment; Martin{nbsp}Juckes; Martin{nbsp}Raspaud; Randy{nbsp}Horne; Timothy{nbsp}Whiteaker; David{nbsp}Blodgett; Charlie{nbsp}Zender; Daniel{nbsp}Lee; David{nbsp}Hassell; Alan{nbsp}D.{nbsp}Snow; Tobias{nbsp}Kölling; Dave{nbsp}Allured; Aleksandar{nbsp}Jelenak; Anders{nbsp}Meier{nbsp}Soerensen; Lucile{nbsp}Gaultier; Sylvain{nbsp}Herlédan; Fernando{nbsp}Manzano; Lars{nbsp}Bärring; Christopher{nbsp}Barker; Sadie{nbsp}Bartholomew
-Version{nbsp}{current-version},{nbsp}{nbsp}{docprodtime}: See{nbsp}https://cfconventions.org{nbsp}for{nbsp}further{nbsp}information.
+Brian Eaton; Jonathan Gregory; Bob Drach; Karl Taylor; Steve Hankin; Jon Blower; John Caron; Rich Signell; Phil Bentley; Greg Rappa; Heinke Höck; Alison Pamment; Martin Juckes; Martin Raspaud; Randy Horne; Timothy Whiteaker; David Blodgett; Charlie Zender; Daniel Lee; David Hassell; Alan D. Snow; Tobias Kölling; Dave Allured; Aleksandar Jelenak; Anders Meier Soerensen; Lucile Gaultier; Sylvain Herlédan; Fernando Manzano; Lars Bärring; Christopher Barker; Sadie Bartholomew
+:revnumber: {current-version}
+:revdate: {docprodtime}
 :doctype: book
 :pdf-folio-placement: physical
 :sectanchors:
 :toc: macro
 :toclevels: 3
 
+<<<
+Climate and Forecast Conventions version {current-version} {doi-text}
+
+image:images/cc-zero.svg[, 10%]
+This document is dedicated to the public domain following the link:https://creativecommons.org/publicdomain/zero/1.0/[Creative Commons Zero v1.0 Universal] Deed.
+
+The Climate and Forecasting Conventions website https://cfconventions.org/ contains additional resources and provides further information.
+ifdef::final[]
+ +
+ +
+ _Use the following reference to cite this version of the document:_ +
+Eaton, B., Gregory, J., Drach, B., Taylor, K., Hankin, S. et al. (2023). NetCDF Climate and Forecast (CF) Metadata Conventions ({current-version}). CF Community. https://doi.org/10.5072/zenodo.130529
+endif::[]
+ifndef::final[]
+ +
+ +
+ _DON’T use the following reference to cite this version of the document, as it is only shown as a draft:_ +
+Eaton, B., Gregory, J., Drach, B., Taylor, K., Hankin, S. et al. (2023). NetCDF Climate and Forecast (CF) Metadata Conventions ({current-version}). CF Community. https://doi.org/10.5072/zenodo.130529
```

Comment:
I think that it would be more correct to direct users to the most recent *published* version if they do want to cite the CF Conventions.

Reply:
Yes, but I have adopted @JonathanGregory suggestion to uses an non looking genuine DOI untile a new DOI is been drafted and pre-reserved